src/main/java/com/ncc/aif/ValidateAIF.java

```diff
@@ -139,86 +143,41 @@ public static void main(String[] args) throws IOException {
     }
 
     /**
-     * Returns whether or not the KB and hypotheses are valid
-     */
-    public boolean validateTA3(Model dataToBeValidated) {
```

Comment:
If we're removing this method, we should probably also remove the ta2ShaclModel and the underlying aida_hypothesis.shacl file

Reply:
Removed.